web/src/beta/features/ProjectSettings/hooks.ts

```diff
@@ -0,0 +1,175 @@
+      await useUpdateProject({ projectId, ...settings });
+    },
+    [projectId, useUpdateProject],
+  );
+
+  const handleArchiveProject = useCallback(
+    async (archived: boolean) => {
+      const { status } = await useArchiveProject({ projectId, archived });
+      if (status === "success") {
+        navigate(`/settings/workspaces/${workspaceId}/projects`);
+      }
+    },
+    [workspaceId, projectId, useArchiveProject, navigate],
+  );
+
+  const handleDeleteProject = useCallback(async () => {
+    const { status } = await useDeleteProject({ projectId });
+    if (status === "success") {
+      navigate(`/settings/workspaces/${workspaceId}/projects`);
+    }
+  }, [workspaceId, projectId, useDeleteProject, navigate]);
+
+  const handleUpdateProjectBasicAuth = useCallback(
+    async (settings: PublicBasicAuthSettingsType) => {
+      if (!projectId) return;
+      await useUpdateProjectBasicAuth({ projectId, ...settings });
+    },
+    [projectId, useUpdateProjectBasicAuth],
+  );
+
+  const handleUpdateProjectAlias = useCallback(
+    async (settings: PublicAliasSettingsType) => {
+      if (!projectId) return;
+      await useUpdateProjectAlias({ projectId, ...settings });
+    },
+    [projectId, useUpdateProjectAlias],
+  );
+
+  const stories = useMemo(() => scene?.stories ?? [], [scene?.stories]);
+  const currentStory = useMemo(
+    () =>
+      tab === "story"
+        ? stories.find(s => s.id === subId) ?? stories[0]
+        : tab === "public"
+        ? stories.find(s => s.id === subId)
+        : undefined,
+    [tab, subId, stories],
+  );
+
+  const { useUpdateStory } = useStorytellingAPI();
+  const handleUpdateStory = useCallback(
+    async (settings: PublicSettingsType & StorySettingsType) => {
+      if (!scene?.id || !currentStory?.id) return;
+      await useUpdateStory({ storyId: currentStory.id, sceneId: scene.id, ...settings });
+    },
+    [useUpdateStory, currentStory?.id, scene?.id],
+  );
+
+  const handleUpdateStoryBasicAuth = useCallback(
+    async (settings: PublicBasicAuthSettingsType) => {
+      if (!scene?.id || !currentStory?.id) return;
+      await useUpdateStory({ storyId: currentStory.id, sceneId: scene.id, ...settings });
+    },
+    [useUpdateStory, currentStory?.id, scene?.id],
+  );
+  const handleUpdateStoryAlias = useCallback(
+    async (settings: PublicAliasSettingsType) => {
+      if (!scene?.id || !currentStory?.id) return;
+      await useUpdateStory({ storyId: currentStory.id, sceneId: scene.id, ...settings });
+    },
+    [useUpdateStory, currentStory?.id, scene?.id],
+  );
+
+  const { getAccessToken } = useAuth();
+  const [accessToken, setAccessToken] = useState<string>();
+
+  useEffect(() => {
+    getAccessToken().then(token => {
+      setAccessToken(token);
+    });
+  }, [getAccessToken]);
+
+  const extensions = useMemo(
+    () => ({
+      library: window.REEARTH_CONFIG?.extensions?.pluginLibrary,
```

Comment:
Please access the config with the config function exported from config settings 🙏🏻

Reply:
Updated.